xp/docker-compose.sxa.yml

```diff
@@ -5,6 +5,7 @@ services:
     volumes:
       - .\sitecore\sxa:C:\sxa
       - ..\files:C:\files-mount
+      - ..\files:C:\inetpub\wwwroot\sitecore\App_Data\packages
```

Comment:
This is a bit hacky but the install module task requires the package.zip to be present here.